api/src/routes.ts

```diff
@@ -29,5 +29,13 @@ export const routes: IRoute[] = [
     { path: "/milestones/:network", method: "get", folder: "milestones", func: "get" },
     { path: "/stats/:network", method: "get", folder: "stats", func: "get", sign: true },
     { path: "/market/:currency", method: "get", folder: "market", func: "get", sign: true },
-    { path: "/did/:network/:did", method: "get", folder: "identity/resolution", func: "get" }
+    { path: "/did/:network/:did", method: "get", folder: "identity/resolution", func: "get" },
```

Comment:
Can we update the endpoints so that they all belong in the did namespace
e.g.
`/did-history/:network/:did` =>  `/did/:network/:did/history`
`/diff-history/:network/:integrationMsgId` => `/did/:network/diff/:integrationMsgId`